cardano-db-sync/src/Cardano/DbSync/Era/Byron/Insert.hs

```diff
@@ -237,24 +237,37 @@ insertByronTx syncEnv blkId tx blockIndex = do
   disInOut <- liftIO $ getDisableInOutState syncEnv
   if disInOut
     then do
-      void . lift . DB.insertTx $
-        DB.Tx
-          { DB.txHash = Byron.unTxHash $ Crypto.serializeCborHash (Byron.taTx tx)
-          , DB.txBlockId = blkId
-          , DB.txBlockIndex = blockIndex
-          , DB.txOutSum = DbLovelace 0
-          , DB.txFee = DbLovelace 0
-          , DB.txDeposit = Nothing -- Byron does not have deposits/refunds
-          -- Would be really nice to have a way to get the transaction size
-          -- without re-serializing it.
-          , DB.txSize = fromIntegral $ BS.length (serialize' $ Byron.taTx tx)
-          , DB.txInvalidHereafter = Nothing
-          , DB.txInvalidBefore = Nothing
-          , DB.txValidContract = True
-          , DB.txScriptSize = 0
-          }
+      txId <-
+        lift . DB.insertTx $
+          DB.Tx
+            { DB.txHash = Byron.unTxHash $ Crypto.serializeCborHash (Byron.taTx tx)
+            , DB.txBlockId = blkId
+            , DB.txBlockIndex = blockIndex
+            , DB.txOutSum = DbLovelace 0
+            , DB.txFee = DbLovelace 0
+            , DB.txDeposit = Nothing -- Byron does not have deposits/refunds
+            -- Would be really nice to have a way to get the transaction size
+            -- without re-serializing it.
+            , DB.txSize = fromIntegral $ BS.length (serialize' $ Byron.taTx tx)
+            , DB.txInvalidHereafter = Nothing
+            , DB.txInvalidBefore = Nothing
+            , DB.txValidContract = True
+            , DB.txScriptSize = 0
+            }
+
+      when (ioTxCBOR iopts) $ do
+        void
+          . lift
+          . DB.insertTxCBOR
+          $ DB.TxCbor
+            { DB.txCborTxId = txId
+            , DB.txCborBytes = serialize' tx
```

Comment:
Don't think so as there's `instance ToCBOR TxAux` but I've switched this to do `serialize' $ Byron.taTx tx` similar to TX size calculation above. Looks like this will omit witnesses (and annotations) but Byron TXs are not that important anyway and there's [another issue for storing TX wits]( #1193) which I've might try to tackle at some point.